test/dotnet-new3.UnitTests/Assertions/CommandResultAssertions.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
I believe this are extractions from framework we discussed this week. Does it make sense to take them to separate assembly?
- it can be reused in other tests
- if we discover that this assembly is shippable, we might replace it with reference